### PR TITLE
I had http and https backwards

### PR DESCRIPTION
**Pull request by robertabbott**
sorry

### CODE REVIEW

**Comment by anthonybishopric** [comment]
:+1: I think we can improve these two status related fields significantly - the naming isn't entirely obvious. `StatusHTTP` being false could be assumed to mean there are no checks at all.

File: integration/single-node-slug-deploy/check.go [hunk 1]
```diff
@@ -300,7 +301,6 @@ func postHelloManifest(dir string) error {
 	manifest := &pods.Manifest{}
 	manifest.Id = "hello"
 	manifest.StatusPort = 43770
-	manifest.StatusHTTP = true
```
Comment (anthonybishopric) [inline integration/single-node-slug-deploy/check.go]: So how was this passing before? Hello should have been listening on HTTPs before?

File: pkg/watch/health.go [hunk 2]
```diff
@@ -166,9 +166,9 @@ func updatePods(store kp.Store,
 				Client: client,
 			}
 			if man.Manifest.StatusHTTP {
-				sc.URI = fmt.Sprintf("https://%s:%d/_status", node, man.Manifest.StatusPort)
-			} else {
 				sc.URI = fmt.Sprintf("http://%s:%d/_status", node, man.Manifest.StatusPort)
+			} else {
+				sc.URI = fmt.Sprintf("https://%s:%d/_status", node, man.Manifest.StatusPort)
```
Comment (bwester) [inline pkg/watch/health.go]: :+1:

File: integration/single-node-slug-deploy/check.go (hunk 1 above)
Comment (robertabbott) [inline integration/single-node-slug-deploy/check.go]: The tests and the watch pkg were both backwards. Check was listening on https and when statusHTTP was true the watch pkg was using https

**Comment by bwester** [comment]
:ship:

**Pull request merged**